tests/net/neighbor/src/neighbor.c

```diff
@@ -359,13 +358,17 @@ static bool run_tests(void)
 }
 
 
-void main(void)
+void main_neighbor(void)
```

Comment:
rename this, do not call this main_<something>, use the type of test you are doing as a name.